djangocms_versioning_filer/helpers.py

```diff
@@ -95,10 +93,16 @@ def filename_exists(request, folder_id=None):
             upload = list(request.FILES.values())[0]
             filename = upload.name
         else:
-            filename = request.GET.get('qqfile', False) or request.GET.get('filename', False) or ''
+            # TODO: Workaround for testing. Need to see why file upload through the GUI uses POST and unittest use GET
+            try:
+                filename = request.POST['file']
+            except:
```

Comment:
Could this be changed so it isn't an open exception?

Reply:
Fixed